python/tvm/contrib/target/onnx.py

```diff
@@ -662,6 +662,86 @@ def convert_attributes(cls, attrs):
         return {"to": getattr(TensorProto, attrs.dtype.upper())}
 
 
+class Resize(OpConverter):
+    """Operator converter for Resize."""
+
+    @classmethod
+    def convert_attributes(cls, attrs):
+        method = attrs.get_str("method")
+        if method == "nearest_neighbor":
+            mode = b"nearest"
+        elif "linear" in method:  # linear / bilinear
+            mode = b"linear"
+        elif "cubic" in method:  # cubic / bicubic
+            mode = b"cubic"
```

Comment:
It might be good to add an `else` to raise an error for unsupported values.

Reply:
Done